commit 772d61f3ed762a50ea4436b7fb70e7024674e6c6
Author: Nicholas Marriott <nicm@openbsd.org>
Date:   Wed May 15 15:32:14 2013 +0000

    RIS should reset focus reporting, from Hayaki Saito.

diff --git a/screen-write.c b/screen-write.c
index c0935c9..7fcfc5e 100644
--- a/screen-write.c
+++ b/screen-write.c
@@ -56,7 +56,7 @@ screen_write_reset(struct screen_write_ctx *ctx)
 	screen_reset_tabs(s);
 	screen_write_scrollregion(ctx, 0, screen_size_y(s) - 1);
 
-	s->mode &= ~(MODE_INSERT|MODE_KCURSOR|MODE_KKEYPAD);
+	s->mode &= ~(MODE_INSERT|MODE_KCURSOR|MODE_KKEYPAD|MODE_FOCUSON);
 	s->mode &= ~(ALL_MOUSE_MODES|MODE_MOUSE_UTF8|MODE_MOUSE_SGR);
 
 	screen_write_clearscreen(ctx);
